### ISSUE TITLE
Remove all references to Namecoin

### ISSUE THREAD

**Issue by verdulo** [chronokings/huntercore#1]
Please remove all references to Namecoin - currently you cannot run Namecoin Core (which is the default downloadable version since march) and Hunteroin on the same system, because they use the same registry keys, and they try to operate on the same files (wallet, blockchain, etc).

**Comment by domob1812** [comment]
I've not had any problems so far myself, but I usually run the non-Qt versions (i. e., `namecoind` and `huntercoind`).  AFAIK, those have no such issues.  As a workaround, you can also explicitly set `-datadir` if it conflicts.

But of course, we should try to fix any wrong references.

**Comment by p3yot3** [comment]
I've never been able to merge mine HUC & NMC at the same time with p2pool, despite using explicit -datadir commands (as I do with all my wallets). I always put this down to them being the same code, but it did just occur to me that the chain ID's might be the same?

**Comment by domob1812** [comment]
No, the chain IDs are definitely different (you can check the code yourself if you want).  I'm pretty sure that F2Pool did in the past merge-mine both NMC and HUC just fine.  (But I don't know anything about P2Pool, except how the idea roughly works.)
